prov/efa/src/rdm/rxr_msg.h

```diff
@@ -55,7 +55,7 @@ void rxr_msg_update_peer_rxe(struct fi_peer_rx_entry *peer_rxe,
 	 * We will do the second way, see efa_rdm_srx_free_entry().
 	 */
 	peer_rxe->flags = (rxe->fi_flags & ~FI_MULTI_RECV);
-	if (rxe->desc && rxe->desc[0] && ((struct efa_mr *)rxe->desc[0])->shm_mr) {
```

Comment:
Did coverity complain about`rxe->desc && rxe->desc[0]`

Reply:
Yes `rxe->desc` is an array, so it will never be NULL. comparing it with NULL does not make sense.

Reply:
Never mind. I thought `rxe->desc` is equivalent to `rxe->desc[0]`